lapack/gonum/errors.go

```diff
@@ -66,21 +76,35 @@ const (
 	negDimension  = "lapack: negative matrix dimension"
 	negZ          = "lapack: negative z value"
 	nLT0          = "lapack: n < 0"
+	nLT1          = "lapack: n < 1"
```

Comment:
What is the ordering that's being used here?

Reply:
This is because of `Dlacn2` which uses reverse communication to compute A*x and A^T*x (similarly to `linsolve`) and I didn't see a sensible way in the algorithm to allow `n == 0`. Looking again now I see that I could allow `n == 0` by terminating the iteration loop by directly `return 0, 0` but that deviates from the reference and from the documented behavior. Unlike with more common lapack functions where it may be convenient to relieve the caller from checking the matrix size, here I think it is ok to enforce non-zero problem size.

Reply:
Sorry, I mean the ordering of lines. It seems almost alphabetical.

Reply:
Ah, yes, I will clean and reorder the error messages later.